etc/systemd/system/restic-backup@.service

```diff
@@ -6,5 +6,5 @@ OnFailure=status-email-user@%n.service
 Type=simple
 Nice=10
 ExecStart=/usr/local/sbin/restic_backup.sh
-# $HOME or $XDG_CACHE_HOME must be set for restic to find /root/.cache/restic/
-Environment="HOME=/root"
\ No newline at end of file
+EnvironmentFile=-/etc/restic/_global.env
```

Comment:
What does the `-` mean here?

Reply:
It means that if the file isn't there, it won't error (I looked it up too)

Reply:
I guess it should error, as it's needed? Either way I still think it's nicer to not spread the knowledge of the global around, and contain the configuration from one access point. Makes this file smaller too :)

Reply:
Agreed.

When I was reading about systemd, it may have said that the EnvironmentFile isn't a shell script; it's just variables; it may not be able to run the source command. I'll have to try it.

Reply:
Also, it's not actually needed, necessarily: if all config is specific to the .env file, you don't need globals.

But I agree that making this compatible with manual scripting should be a goal. I need to try out some stuff and see.